content/publish/form.json

```diff
@@ -128,7 +128,7 @@
         "label": "Access Type",
         "help": "Choose how you want your files to be accessible for the specified price.",
         "type": "boxSelection",
-        "options": ["Download", "Compute"],
+        "options": ["Access", "Compute"],
```

Comment:
This should stay as "Download", or more precisely we need to make sure that in the final UI "Download" is displayed. Pretty sure you changed this cause somewhere along the data stack we probably use these values here for some detection

Reply:
we do another title mapping in the form field definitions so all good